remote/remote_state_s3.go

```diff
@@ -390,6 +402,12 @@ func EnableVersioningForS3Bucket(s3Client *s3.S3, config *RemoteStateConfigS3, t
 		VersioningConfiguration: &s3.VersioningConfiguration{Status: aws.String(s3.BucketVersioningStatusEnabled)},
 	}
 	_, err := s3Client.PutBucketVersioning(&input)
+	if config.S3ForcePathStyle && err != nil {
+		terragruntOptions.Logger.Printf("Versioning not supported for bucket %s", config.Bucket)
```

Comment:
How do we know that `err` implies versioning is not supported? Couldn't it be some other type of error too?

Reply:
> How do we know that err implies versioning is not supported? Couldn't it be some other type of error too?

Well, that has been nagging me since the moment I coded it ;). I think the right way is to add a terragrunt internal config option`skip_bucket_versioning` configuration. Then if it fails, user can configure it. Shall I do it that way?

Reply:
Well, that has been nagging me since the moment I coded it ;). I think the right way is to add a terragrunt internal config optionskip_bucket_versioning configuration. Then if it fails, user can configure it. Shall I do it that way?